scripts/bmm_examples/Example2_Loaiza&Souza2018.R

```diff
@@ -74,86 +75,116 @@ levels(wmData$ageGroup) <- c("Young","Old")
 levels(wmData$RI) <- c("short","long")
 levels(wmData$cueCond) <- c("No","Retro")
 
+# specfiy contrasts
+contrasts(wmData$ageGroup) <- bayestestR::contr.equalprior(2)
+contrasts(wmData$RI) <- bayestestR::contr.equalprior(2)
+contrasts(wmData$cueCond) <- bayestestR::contr.equalprior(2)
+
 ###############################################################################!
 # 2) Model estimation ----------------------------------------------------------
 ###############################################################################!
 
+# set up the bmmodel object
+LS_model <- mixture2p(resp_error = "dev_rad")
 
 #' set up the formula for the mixture model.
 #' Although, we do not want to estimate the mean of the two von Mises distributions,
 #' we have to initialize the formula to specify the dependent variable.
 #' Using priors (see step 3), we will constrain the means of both von Mises distributions
 #' to zero. Additionally, we will use priors to fix the precision (kappa) of the
 #' second von Mises to be zero (at least practically zero). 
-LS_mixFormula_bmm <- bf(
-  # Initializing the dependent variable
-  dev_rad ~ 1,
-  
+LS_formula <- bmf(
   # estimating fixed intercept & random intercept for kappa of the first von Mises
-  kappa ~ 0 + ageGroup:RI:cueCond + (0 + RI:cueCond || gr(id, by = ageGroup)), 
+  kappa ~ 0 + ageGroup + ageGroup:RI + ageGroup:cueCond + ageGroup:RI:cueCond  + 
+    (0 + RI + cueCond + RI:cueCond || gr(id, by = ageGroup)), 
   
   # estimating fixed intercept & random intercept for the mixing proportion 
   # of the target vonMises (i.e., p_mem)
-  thetat ~ 0 + ageGroup:RI:cueCond + (0 + RI:cueCond || gr(id, by = ageGroup))
+  thetat ~ 0 + ageGroup + ageGroup:RI + ageGroup:cueCond + ageGroup:RI:cueCond  + 
+    (0 + RI + cueCond + RI:cueCond || gr(id, by = ageGroup))
 )
 
-# constrain parameters using priors
-mixPriors <- prior(normal(0,0.5), class = "b", dpar = "theta1") +
-  prior(normal(0,0.5), class = "b", dpar = "kappa1")
+# check the default priors
+default_prior(LS_formula, data = wmData, model = LS_model)
+
+prior <- 
```

Comment:
something is missing here

Reply:
Deleted. This was left over from the bug in the `default_prior` function when I specified user priors to fit the model. I refit the model with the now correctly passed `default_priors`